openedx/core/djangoapps/content/course_overviews/management/commands/generate_course_overview.py

```diff
@@ -49,4 +47,4 @@ def handle(self, *args, **options):
             if not course_keys:
                 log.fatal('No courses specified.')
```

Comment:
Log fatal doesn't have a side effect of raising an error.  It just indicates the logging level.
What do you mean from the "same location"?